From c0044ed67f0102dcfe2ffeef39c8e7d0eb782301 Mon Sep 17 00:00:00 2001
From: shamim ali <ashamim@ami.com>
Date: Mon, 23 Jun 2025 10:46:29 -0400
Subject: [PATCH] Added last entryId for ipmi

This patch will add last entryId in dbus ,
so ipmi will sync with phsphor-logging after roll
over.

Signed-off-by: shamim ali <ashamim@ami.com>
---
 include/defaults.hpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/defaults.hpp b/include/defaults.hpp
index 5b941df..996383a 100644
--- a/include/defaults.hpp
+++ b/include/defaults.hpp
@@ -639,6 +639,8 @@ setting->addProperty("RetryIntervalMS", static_cast<uint8_t>(20));
     setting->addProperty("ErrorFlags", std::vector<std::pair<std::string, bool>>{
 	    {"default", false},{"ipmi", false},{"raid", false}});
 
+    setting->addProperty("lastEntryId", static_cast<uint16_t>(0));
+
 
     setting = &settings.emplace_back(objectServer,
                                      "/xyz/openbmc_project/pfr/last_events",
-- 
2.34.1

